<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegLinealMulti</class>
 <widget class="QMainWindow" name="RegLinealMulti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegLinealMulti</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>x1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>x2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_12"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_13"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_14"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_15"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_16"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_17"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_8"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_9"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_10"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_11"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_18"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Metodo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Escape</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>284</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
